### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by hiroto-udagawa**

### CODE REVIEW

**Comment by janewliang** [comment]
You gotta get rid of "sudo apt-get upgrade" in travis.yml.

**Comment by benjaminleroy** [comment]
You also didn't move the convolution commentary to the misc folder :(

**Comment by hiroto-udagawa** [comment]
just updated

**Comment by benjaminleroy** [comment]
Could you add the commentary file to misc? Also, I worked on your final model selection thing a good amount, iono if you want to not submit it (see my push anyway)

**Comment by hiroto-udagawa** [comment]
i think i did add the commentary file

**Comment by reychil** [comment]
in smooth_final.py, line 80 - Where is fwhm used after we define it? If you wanted to use it, it should be used as the argument for sigma in the smoothing function

File: final/scripts/smooth_final.py [hunk 1]
```diff
@@ -4,22 +4,20 @@
 Imports functions from multiple locations, see test files in correct folders
 
 Potential variants:
-	- this currently uses sigma =1.5/ fwhm =3.5, see lines 90-92
```
Comment (benjaminleroy) [inline final/scripts/smooth_final.py:7]: remove the stuff about fwhm, note that fwhm = (2*np.sqrt(2 *np.log(2))) \* sigma,
and the original paper used much smoother data (with fwhm as 5.5 mm)

File: final/scripts/smooth_final.py [hunk 2]
```diff
@@ -90,7 +76,7 @@
 	for time in np.arange(data.shape[-1]):
 		# Kind of arbitrary chosen time
 
-		sigma = 1.5
+		sigma = 1
 		fwhm = (2*np.sqrt(2 *np.log(2))) * sigma
```
Comment (benjaminleroy) [inline final/scripts/smooth_final.py]: remove this fwhm line (it isn't used in the code)

**Comment by benjaminleroy** [comment]
Good job. I'm glad we talked about all the updates,

**Pull request merged**